src/ray/raylet/worker_pool.cc

```diff
@@ -385,6 +386,16 @@ WorkerPool::BuildProcessCommandArgs(const Language &language,
     }
   }
 
+  if (language == Language::PYTHON && worker_type == rpc::WorkerType::WORKER &&
+      RayConfig::instance().preload_python_modules().size() > 0) {
+    std::string serialized_preload_python_modules =
```

Comment:
nit, but should we just make this config a string and parse & validate from the worker side?